### DISCUSSION TITLE
Update error: git submodule and params.toml

### DISCUSSION THREAD

**Discussion by stockphrase** [jpanther/congo#820] [General]
I'm using a git submodule and attempt to update the theme (from v2.5.2) with:

`git submodule update --remote --merge`

I get an error that local changes to `params.toml` will be overwritten and that I should commit or stash them. 

But I have already added and committed all files. I also tried to create a stash. No matter what I do the error occurs. How do I get past this?

**Comment by wolfspyre** [comment]
in general, one shouldn’t edit the sub module files, but rather override them in your main project.

this allows the sub module to remain in a pristine condition so you can update easily.

check out the hugo docs for overriding theme defaults… its fairly well documented there…
